app/src/main/java/com/ultratechies/ghala/ui/auth/OtpVerificationFragment.kt

```diff
@@ -52,22 +51,18 @@ class OtpVerificationFragment : Fragment() {
 
     private fun fetchOTPlistener() {
         viewLifecycleOwner.lifecycleScope.launch {
-            repeatOnLifecycle(Lifecycle.State.STARTED) {
```

Comment:
Hey @malcolmmaima we need the repeatOnLifecycle  to launch the block in a new coroutine every time the lifecycle is in the STARTED state and to cancel it when it's stopped. I think we need to reference it like this in the fragment viewLifecycleOwner.repeatOnLifecycle(Lifecycle.State.STARTED)

Reply:
@Njumbi does this apply to all other fragments or just OTP ... i'll revert all to previous, although lint is complaining ... may need to investigate that

Reply:
It will apply to all fragments, lets hope if we make that change lint will not complain.

Reply:
@Njumbi check it out now, if all good feel free to merge